client/app/dedicated/housing/dashboard/dedicated-housing-dashboard.html

```diff
@@ -3,161 +3,176 @@
-                    <li class="oui-tile__item">
-                        <div class="oui-tile__definition">
-                            <strong data-i18n-static="housing_name"></strong>
-                            <div>
-                                <span data-ng-show="housing.name"
-                                      data-ng-bind="housing.name">
-                                </span>
-                                <span class="text-muted"
-                                      data-ng-hide="!housing.name"
-                                      data-i18n-static="housing_configuration_name_unavailable">
-                                </span>
-                            </div>
-                        </div>
-                    </li>
-                    <li class="oui-tile__item">
-                        <div class="oui-tile__definition">
-                            <strong class="d-block"
-                                    data-i18n-static="housing_tab_stats_legend_bay">
-                            </strong>
-                            <span data-ng-bind=":: housing.rack"></span>
-                        </div>
-                    </li>
-                    <li class="oui-tile__item">
-                        <div class="oui-tile__definition">
-                            <strong class="d-block"
-                                    data-i18n-static="housing_tab_stats_securitycode">
-                            </strong>
-                            <span data-ng-bind=":: housing.securityCode"></span>
-                        </div>
-                    </li>
-                    <li class="oui-tile__item">
-                        <div class="oui-tile__definition">
-                            <dt>
-                                <service-expiration-label
-                                        service-infos="housing.serviceInfos">
-                                </service-expiration-label>
-                            </dt>
-                            <dd>
-                                <service-expiration-date
-                                        service-name="housing.name"
-                                        service-type="'DEDICATED_HOUSING'"
-                                        service-infos="housing.serviceInfos">
-                                </service-expiration-date>
-                            </dd>
-                        </div>
-                    </li>
-                    <li class="oui-tile__item">
-                        <div class="oui-tile__definition">
-                            <dt data-i18n-static="housing_creation_date"></dt>
-                            <dd data-ng-bind="housing.creation | date:'mediumDate'"></dd>
-                        </div>
-                    </li>
-                </ul>
+                        </dd>
+                    </dl>
+                </div>
+                <div class="oui-tile__item">
+                    <dl class="oui-tile__definition">
```

Comment:
I don't know if it's related to the style you want to apply but in fact you are creating some `<dl>` lists that always have a single item in it.